liblangutil/EVMVersion.h

```diff
@@ -39,7 +39,7 @@ class EVMVersion:
 	boost::equality_comparable<EVMVersion>
 {
 public:
-	EVMVersion() {}
+	EVMVersion() = default;
```

Comment:
it's actually needed to enable default initialization of private members. (since all member variables are private, we have to have a default constructor in order to be able to write something like `EVMVersion version;` (otherwise the caller-side can't initialize.)

Reply:
@christianparpart I'm pretty sure that's not true either. Try this, it'll work:

```
#include <iostream>
class A {
public:
    A() { std::cout << "A::A()" << std::endl; }
};
class B {
private:
    A a;
};

int main()
{
    B b;
}
```

Reply:
For reference see ``Implicitly-declared default constructor`` (https://en.cppreference.com/w/cpp/language/default_constructor)